sklearn/metrics/tests/test_classification.py

```diff
@@ -248,6 +248,15 @@ def test_matthews_corrcoef_nan():
     assert_equal(matthews_corrcoef([0, 0], [0, 1]), 0.0)
 
 
+def test_matthews_corrcoef_result():
+    rng = np.random.RandomState(0)
+    y_true = rng.randint(0, 2, size=20)
+    y_pred = rng.randint(0, 2, size=20)
+
+    assert_almost_equal(matthews_corrcoef(y_true, y_pred),
+                        np.corrcoef(y_true, y_pred)[0, 1], 10)
```

Comment:
@jnothman Doesn't this test take care of that? ( atleast for the non weighted case? )